pkg/apis/pingcap.com/v1alpha1/types.go

```diff
@@ -432,12 +462,15 @@ type BackupScheduleStatus struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// +k8s:openapi-gen=true
 // Restore represents the restoration of backup of a tidb cluster.
 type Restore struct {
-	metav1.TypeMeta   `json:",inline"`
+	metav1.TypeMeta `json:",inline"`
+	// +k8s:openapi-gen=false
 	metav1.ObjectMeta `json:"metadata"`
 
-	Spec   RestoreSpec   `json:"spec"`
+	Spec RestoreSpec `json:"spec"`
+	// +k8s:openapi-gen=false
 	Status RestoreStatus `json:"status"`
 }
 
```

Comment:
No need to add "// +k8s:openapi-gen=true" for RestoreList?

Reply:
I missed it.